ui/app/adapters/variable.js

```diff
@@ -13,7 +13,7 @@ export default class VariableAdapter extends ApplicationAdapter {
     let data = this.serialize(snapshot);
     let baseUrl = this.buildURL(type.modelName, data.ID);
     const checkAndSetValue = snapshot?.attr('modifyIndex') || 0;
-    return this.ajax(`${baseUrl}&cas=${checkAndSetValue}`, 'PUT', { data });
+    return this.ajax(`${baseUrl}?cas=${checkAndSetValue}`, 'PUT', { data });
```

Comment:
`praise`: simple solution!

Reply:
`suggestion`: This probably should've gotten tested when it was originally implemented. Missing this logic in our Mirage set-up caused our tests to pass when they shouldn't have. I think another reason why using Playwright tests would've caught this error.